app/views/idv/cac/choose_method.html.erb

```diff
@@ -1,24 +1,45 @@
 <%= render 'shared/maintenance_window_alert' do %>
+  <style>
+    .flex-container {
+        display: flex;
+    }
+    @media (max-width: 30em) {
+        .flex-container {
+            flex-direction: column;
+        }
+    }
+  </style>
   <h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
-  <%= t('cac_proofing.info.choose_method') %>
+  <p><%= t('cac_proofing.info.choose_method') %></p>
+  <p><%= t('cac_proofing.info.disclaimer') %></p>
+  <div class="flex-container mt3 mb3">
+    <%= image_tag asset_url('PIV-CAC-round.svg'), width: 120, height: 116, class: 'mr3 mb2'%>
```

Comment:
Not sure if we'd expect any issues with case sensitivity of OS, but I notice most (all?) other files in the repository are lower-cased, i.e. `piv-cac-round.svg`.

Reply:
Yeah, it was just the file name that annie had named it. I lowercased it.